src/librustc_resolve/diagnostics.rs

```diff
@@ -93,6 +93,23 @@ mod bar {
 ```
 "##,
 
+E0253: r##"
+Attempt was made to import an unimportable value. This can happen when
+trying to import a method from a trait. An example of this error:
+
+```
+mod foo {
+    pub trait MyTrait {
+        fn do_something();
+    }
+}
+use foo::MyTrait::do_something;
+```
+
+In general, it's not legal to directly import methods belonging to a
```

Comment:
No exceptions as far as I'm aware.  Since this is mid-commit I can always make another PR to fix it up.